modules/administration-guide/pages/removing-user-data.adoc

```diff
@@ -30,7 +33,9 @@ Following this procedure makes the service compliant to EU General Data Protecti
 //   'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
 // ----
 
-. Remove the __<id>__ user data that {prod-short} server manages, such as user preferences: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/remove, click btn:[Try it out], set *id*: __<id>__, and click btn:[Execute]. Expect a `204` response code:
+. Remove the __<id>__ user data that {prod-short} server manages, such as user preferences:
```

Comment:
@dmytro-ndp, do you happen to know if we can use `https://<che_fqdn>/api` to remove user data?
Here's the current version of the docs:
https://www.eclipse.org/che/docs/stable/administration-guide/removing-user-data/
Here's Jana's preview:
https://63eb684ebf50410b07969003--eclipse-che-docs-pr.netlify.app/docs/next/administration-guide/removing-user-data/

Reply:
It would better to ask @ScrewTSW who is testing user data removal, or @ibuziuk .